packages/jest-mock/src/index.ts

```diff
@@ -867,19 +1053,23 @@ class ModuleMockerClass {
       }
 
       descriptor[accessType] = this._makeComponent({type: 'function'}, () => {
-        // $FlowFixMe
+        if (!descriptor) {
```

Comment:
we're in a closure so technically this can be altered, but will it really?

Reply:
I'm fine keeping the ignores - I think people would have complained by now if this was actually an issue?